strawberry/field.py

```diff
@@ -274,6 +274,15 @@ async def await_result(result):
         _resolver._is_default = not self.base_resolver  # type: ignore
         return _resolver
 
+    def _get_return_type(self):
+        if self.is_list:
+            type_ = List[self.child._get_return_type()]
```

Comment:
worth nothing that this in future will change to StrawberryList 😊

Reply:
Or potentially `StrawberryType` with `.is_list == True`. I'm still waffling on the implementation...

Reply:
ok! no rush, I think this will be only used internally for now 😊